src/mpags-cipher.cpp

```diff
@@ -4,20 +4,24 @@
 #include "PlayfairCipher.hpp"
 #include "ProcessCommandLine.hpp"
 #include "TransformChar.hpp"
+#include "VigenereCipher.hpp"
```

Comment:
You can now drop the includes of the specific ciphers (Caesar, Playfair, Vigenere) because they are no longer directly used here - everything is abstracted away inside the cipher factory function.